config/README.md

```diff
@@ -6,5 +6,5 @@ This is a central place for Kubernetes-project specific configs for other tools
 
 Directory structure:
 
-[jobs/](./jobs) : job configs for Kubernetes Prow deployment, potentially testgrid configs as well
+[jobs/](./jobs) : job configs for Kubernetes Prow deployment, potentially testgrid configs as well  
```

Comment:
these were pretty ugly all on one line which was not the intention, added two spaces to actually create the newline..